lib/IAMClient.js

```diff
@@ -52,6 +53,19 @@ class IAMClient {
         }
         this.accessKey = accessKey;
         this.secretKeyValue = secretKeyValue;
+        this.log = new Logger('IAMClient', {
+            level: 'info',
```

Comment:
Thought that would be easier than adding another param `config` to the constructor.